src/ProjectOrigin.VerifiableEventStore.Tests/Repository/AbstractTransactionRepositoryTests.cs

```diff
@@ -271,7 +300,48 @@ public async Task CanCreateSeriesOfBlocks()
         block3!.Header.PreviousPublicationHash.Should().BeEquivalentTo(SHA256.HashData(pub2.ToByteArray()));
         block3!.TransactionHashes.Should().HaveCount(transactionList3.Count);
         block3!.TransactionHashes.Should().ContainInOrder(transactionList3.Select(x => x.TransactionHash));
-        var pub3 = new ImmutableLog.V1.BlockPublication();
+        var pub3 = new Registry.V1.BlockPublication();
         await Repository.FinalizeBlock(BlockHash.FromHeader(block3!.Header), pub3);
     }
+
+    [Fact]
+    public async Task CanGetBlocks()
```

Comment:
This test does a lot more than just getting blocks. You are asserting upon its headers, and contents as well. I suggest splitting it up in different tests, or renaming the test, to better reflect, what we are trying to prove.